src/components/DonutChart.js

```diff
@@ -213,12 +210,13 @@ const DonutChart = React.createClass({
 
   render () {
     const position = 'translate(' + this.props.width / 2 + ',' + this.props.height / 2 + ')';
+    const fontSize = Math.min(this.props.width, this.props.height) * 0.2 + 'px';
 
     return (
-      <div style={[styles.component, this.props.style, { height: this.props.height, width: this.props.width }]}>
+      <div style={[styles.component, this.props.style, { fontSize, height: this.props.height, width: this.props.width }]}>
         {this._renderDataLabel()}
         <svg height={this.props.height} width={this.props.width}>
-          <g style={styles.pointer} transform={position}>
+          <g style={{ cursor: this.props.animateOnHover ? 'pointer' : 'auto' }} transform={position}>
```

Comment:
yes, but by default we don't do anything with that click event. So it's misleading to a user if they see a `pointer` cursor, but clicking doesn't do anything. We should leave it up to the person implementing the component to determine if a `pointer` is displayed.